terminology.md

```diff
@@ -261,17 +261,23 @@ A realm consists of a set of [intrinsic](#Intrinsic) objects, an ECMAScript glob
 
 ### Intrinsic
 #### Definition
-Built-in objects that are explicitly referenced by the algorithms of the ECMA262 specification.
+Built-in objects (usually realm-specific) that are explicitly referenced by the algorithms of the ECMA262 specification.
```

Comment:
The function object `Array.prototype.push` is not an object referenced by any ECMA262 algorithm. But it is primordial. IIUC @allenwb 's suggested terms, it is also an intrinsic.

All intrinsics are realm specific. We don't need the "usually"

Reply:
Hmm, I don't want this document to contradict the [spec](https://tc39.es/ecma262/#sec-well-known-intrinsic-objects), which includes both "explicitly referenced by" and "usually". Presumably only the non-primordial intrinsics could be shared across realms, because identity must be distinct for observable cross-realm values. What would you think about text similar to this in both places?

> Built-in values that are explicitly specified by ECMA262. Where observable (i.e., as primordial), intrinsic values are realm-specific.

(where "_values_" pulls in well-known symbols and "_or referenced by_" unambiguously includes implementation-dependent and therefore arguably not "defined" cases like %Number.prototype.toLocaleString%)